app/assets/javascripts/discourse/app/components/footer-nav.gjs

```diff
@@ -14,6 +14,15 @@ export default class FooterNav extends Component {
   @service composer;
   @service modal;
   @service historyStore;
+  @service router;
+
+  EXCLUDE_IN_ROUTES = [
```

Comment:
With https://github.com/discourse/discourse/pull/34522, would it make sense to move this constant to a central place with a clearer name?